aws/dms/tasks.yml

```diff
@@ -76,6 +75,7 @@ cron:
 - dashboard.stages_standards
 - dashboard.standard_categories
 - dashboard.frameworks
+- dashboard.user_storage_ids
```

Comment:
`pegasus.user_storage_ids` was renamed to `dashboard.user_project_storage_ids`

Reply:
Thank you for pointing these out; I've updated the code.